src/main/common/typeconversion.c

```diff
@@ -188,7 +188,7 @@ char *ftoa(float x, char *floatString)
 
     dpLocation = strlen(intString2) - 3;
 
-    strncpy(floatString, intString2, dpLocation);
+    memcpy(floatString, intString2, dpLocation);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Does not check for buffer overflows when copying to destination (CWE-120).  Make sure destination can always hold the source data. ](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=5204956)

Reply:
for the records, this is non-issue due `ftoa` as Pawel reported.